### ISSUE TITLE
boltctl: security field is misleading

### ISSUE THREAD

**Issue by gicmo** [gicmo/bolt#37]
```sh
 ● Dell Thunderbolt Cable
[...]
   ├─ status:      authorized
   │  └─ security: secure
   └─ stored:      yes
      ├─ policy:   auto
      └─ key:      yes (new)
```
The `security` above is the security level of the associated domain controller, not the security of the device. I am thinking of showing the actual authorization of the device (something like `user`, `secure`, `new-key`). This could also be shown on the status line instead, i.e. `authorized (user)`.

**Comment by ybernat** [comment]
1. Just to make sure we are on the same page:
The sysfs attribute of "security" for a device means "the max SL supported by the device" (usually SL2 but old devices may support only SL1, no support for saving a key in those).

1. What are you going to show for "actual authorization of the device" if the device was approved with a key but later the user removed it from ACL? Will the answer changed if the daemon gets restarted?

**Comment by gicmo** [comment]
> The sysfs attribute of "security" for a device means "the max SL supported by the device"

Yeah we are on the same page, `security` here means the maximum security level possible. Hence why I think that currently it might be confusing, because it could be mistaken for the actual security level. The latter is already encoded in the `Status` D-Bus property but currently not printed (see the [`BoltStatus`](https://github.com/gicmo/bolt/blob/master/common/bolt-enums.h#L27) enum).
One thing though (to avoid a misunderstanding): boltd looks up the maximum security level for a device by going to the domain of the device because (at least in my tests, but [switch.c](https://github.com/torvalds/linux/blob/5bbcc0f595fadb4cac0eddc4401035ec0bd95b09/drivers/thunderbolt/switch.c#L943) seems to agree) the device sysfs entry itself seems not have the `security` file -- but the domain has.

> What are you going to show for "actual authorization of the device" if the device was approved with a key but later the user removed it from ACL? Will the answer changed if the daemon gets restarted?

On the D-Bus is would mean that the `Status` property is still `AUTHORIZED_SECURE` (if the kew was new then `AUTHORIZED_NEWKEY`) but the `Stored` property will be `False` and `Key` will be `KEY_MISSING`.
Can already be tested via `boltctl enroll XXX` followed by `boltctl forget XXX`
